app/renderer/components/navigation/navigator.js

```diff
@@ -150,6 +152,172 @@ class Navigator extends React.Component {
     ipc.off(messages.SHORTCUT_ACTIVE_FRAME_FORWARD, this.onForward)
   }
 
+  // BEM Level: navigator__menuBarAndNavigationBar__navigationBarWrapper
+  get navigationBarWrapper () {
```

Comment:
> Because the implications for this (pulling JSX/HTML out to methods) could be serious, I think we should discuss more

Let's do so with another issue 👍